projects/WeTek_Hub/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Debug Shell on /dev/ttyS0
+DefaultDependencies=no
+ConditionKernelCommandLine=console=ttyS0,115200
+
+[Service]
+WorkingDirectory=/storage
+Environment="ENV=/etc/profile"
+ExecStartPre=/bin/sh -c 'echo -en "\033[?25h"'
```

Comment:
you dont need this. cursor was never hidden before so you dont have to force-show it.